src/cdk/drag-drop/drag-ref.ts

```diff
@@ -1097,7 +1097,9 @@ export class DragRef<T = any> {
         const handler = ((event: TransitionEvent) => {
           if (!event || (_getEventTarget(event) === this._preview &&
               event.propertyName === 'transform')) {
-            this._preview.removeEventListener('transitionend', handler);
+            if (this._preview) {
+              this._preview.removeEventListener('transitionend', handler);
```

Comment:
nit: can be reduced to `this._preview?.removeEventListener('transitionend', handler);`